src/ofp_pkt_processing.c

```diff
@@ -1453,14 +1459,16 @@ enum ofp_return_code ofp_packet_input(odp_packet_t pkt,
 		ofp_packet_user_area_reset(pkt);
 	}
 
-	if (ifnet->chksum_offload_flags & OFP_IF_IPV4_RX_CHKSUM)
-		ofp_packet_user_area(pkt)->chksum_flags |=
-			OFP_L3_CHKSUM_STATUS_VALID;
+	if (in_queue != ifnet->loopq_def) {
```

Comment:
It would be good to add a short explanation comment here.

Reply:
ok